### PR TITLE
Draft: Enhance example in crate documentation for requesting UserInfo

### PR DESCRIPTION
**Pull request by benjaminSchilling33**
@ramosbugs 
As promised in https://github.com/ramosbugs/openidconnect-rs/issues/28#issuecomment-770036819, I started extending the example with a UserInfo request.

How are you testing this example?

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -190,6 +190,41 @@
...
+//! impl AdditionalClaims for MyClaims {}
+//!
+//! // Now we create a new user_info request using the client
+//! let userinfo_request = match client.user_info(
+//! AccessToken::new(
+//!   // We re-use the access_token we used for requesting the IdToken
+//!   token_response.access_token().secret().to_string()),
+//!   None,
+//! ) {
+//!   Ok(o) => o,
```
Comment (ramosbugs) [inline src/lib.rs:210]: let's replace this match and the one below with the more common `.map_err(|err| ...)?` pattern

**Review by ramosbugs — COMMENTED** [review]
thanks for the PR! the examples are compile-tested as part of `cargo test --doc`. please take a look at the related CI errors: https://travis-ci.org/github/ramosbugs/openidconnect-rs/jobs/756820151.

since this example is illustrating custom claims and not just the default user info request (with [`CoreUserInfoClaims`](https://docs.rs/openidconnect/2.0.0-alpha.2/openidconnect/core/type.CoreUserInfoClaims.html)), I think it should be separate from the "Getting Started" example. it would be fine to add a `CoreUserInfoClaims` example to Getting Started since it's a common enough need, but let's separate them if you'd like to illustrate how to implement an extension (also a useful example to have!)

File: src/lib.rs (hunk 1 above)
Comment (benjaminSchilling33) [reply src/lib.rs]: How about now?

**Comment by ramosbugs** [comment]
thanks! I merged a slightly simplified version in 6fb05a5153e1b53a7ac543e5db674bc6a9f64cbf

**Pull request closed without merging**